spec/views/collections/_collection.html.erb_spec.rb

```diff
@@ -18,6 +18,11 @@
     expect(rendered).to have_css('h2 a', text: collection.name)
   end
 
+  it 'renders a paragraph with the description of the collection' do
+    render
+    expect(rendered).to have_css('p', text: collection.description)
+  end
+
```

Comment:
Would something like this works?
```ruby
  it 'renders a paragraph with no description of none is given' do
    create(:collection)
    render
    expect(rendered).to have_css('h2 a', count: 2)
    expect(rendered).to have_css('p', count: 1)
  end
```
It doesn't seem to create the second collection though.